kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -314,6 +314,14 @@ object KyuubiConf {
       .stringConf
       .createOptional
 
+  val AUTHENTICATION_LDAP_GUIDKEY: ConfigEntry[String] =
+    buildConf("authentication.ldap.guidKey")
+      .doc("LDAP attribute name whose values are unique in this LDAP server." +
+        "For example:uid or cn.")
+      .version("1.2.0")
```

Comment:
do you want to backport this to 1.2.0 ? the master branch now is 1.3.0

Reply:
and please update the docs with the cmd:
`KYUUBI_UPDATE=1 build/mvn test -DwildcardSuites=org.apache.kyuubi.config.AllKyuubiConfiguration`

Reply:
> do you want to backport this to 1.2.0 ? the master branch now is 1.3.0

This is not a big deal, as we haven't decide the final release of kyuubi yet for the process of Apache incubating

Reply:
I've update the docs